Documents/Functional_Specifications.md

```diff
@@ -0,0 +1,192 @@
+| Milestone 3 | Crete a playable version of the game for a demo | 2023-03-24 |
+| Milestone 4 | Create a final version of the game to present | 2023-04-14 |
+
+### Game Description
+
+Each sugar cube as only one duty, to sweeten a coffee. You will follow the adventures of Sube, a sugar cube, that has been thrown away by mistake and is now trying to go into the coffee. Help Sube navigate through obstacles and reach the coffee cup.
+You will be able to go through countless adventures and levels with Sube all around the house to find the cup and finally sweeten the coffee. Sube will be able to use the environment to his advantage to reach the cup, but be careful, there is things that a sugar should not approach.
+
+
+## Objectives
+
+For this project, the main objectives are:
+- Create a game using Unreal Engine 4.27.
+- Create a game that is fun to play.
+- Create a game that is easy to use.
+- Have a game with original mechanics.
+- Push the limits of our capabilities with Unreal Engine during the period of time we have.
+
+## Requirements
+
+- We must use Unreal Engine 4.27.
+- We must have a playable version of the game by the end of the project.
+- We must have a game that give the player a good experience.
+
+## Functionalities
+
+### Main Menu
+The main menu will be the first menu that the player will see when he starts the game.
+In this menu, the player will have the choice to start the game or to quit the game.
+
+Here is the first version of the main menu from the prototype:
+
+![First Main Menu](https://cdn.discordapp.com/attachments/641190235581054986/1083769795683565738/image.png)
+
+### Pause Menu
+
+The Pause menu will be the menu that will appear when the player will pause the game and will allow the player to resume the game, go to the main menu or quit the game.
+
+Here is the first version of the pause menu from the prototype:
+
+![First Pause Menu](https://cdn.discordapp.com/attachments/641190235581054986/1083769841946738708/image.png)
+
+### Mechanics
+
+The mechanics of the game will be the following:
+
+| Mechanic's name | Description |
+| -------------- | ----------- |
+| **Deadly objects** | The player will have to avoid some obstacles that will kill him if he stays too long close to it. ( e.g. water, heat, etc...) |
+| **Sticky** | The player will have to use the sticky mechanic to be able to stick to allowed walls to cross gaps between places. |
+| **Catapult** | The player will have to use objects that allow him to launch himself. |
+| **Portals** | The player will have to use one-way portals to be able to teleport himself. |
+
+
+## Non-functional Requirements
+
+- The game will be solo-player only.
+- The game will be in 3D.
+- The game will be in English.
+- It will work on Windows.
+
+
+## Personas
+
+<ins>Name:</ins> Jean <br>
+<ins>Age:</ins> 76 <br>
+<ins>Bio:</ins> He retired some months ago but has been wondering how to entertain himself recently and wants to try a new game. <br>
```

Comment:
```suggestion
<ins>Bio:</ins> He's retired since few months ago but have been wondering how to entertain himself recently and wants to try a new game. <br>
```